_posts/2021/2021-12-17-how-the-graphic-design-innovation-lab-boosted-my-creativity.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: How the “Graphic Design Innovation Lab” boosted my creativity
+date: 2021-12-17
+header_image: public/creativity-child.jpg
+header_overlay: true
+header_position: center
+category: events
+tags: ["design", "creativity", "workshop"]
+authors: ["Svenja"]
+about_authors: ["svoelz"]
+---
+
+On the 2nd and 3rd of December, I participated in the [Tomorrow Academy's](https://tomorrowacademy.org/){:target="_blank"} "Graphic Design Innovation Lab" workshop.
+In this post, I would like to share and reflect on some of my impressions and thoughts about the workshop.
+
+## The speakers and the workshop
+
+The twins Thomas and Martin Poschauko are multidisciplinary universal creatives who combine liberal arts, design, creative research, and academic teaching.
+As designers, they work for clients such as Sony Music, Kieler Woche, Deutsche Post, the band Labrassbanda, the Federal Ministry of Finance, or the product design label Auerberg.
+They take care of the graphic design, corporate identity, illustration, and music design.
+The authors of the multiple awards winning book ["Nea Machina - die Kreativmaschine"](https://www.poschauko.de/neamachina/){:target="_blank"} give seminars, hold lectures about "creativity", and explore the creative process in their experiments.
+As creative consultants, they already worked with companies such as MetaDesign, Red Bull, Ferrero, Maloja, Georg Thieme Verlag, Strichpunkt Design, and ServusTV.
+For more information and insights into their work, please visit their [website](https://www.poschauko.de/){:target="_blank"}.
+
+For me, the workshop started a few days before the actual date.
+I received a big package with everything I needed for the workshop, and some great little gifts, such as snacks and drinks.
+
+{% image_custom image="/assets/img/pages/blog/images/material-graphic-design-innovation-lab.png" width="35"%}
+
+During the workshop, the Poschaukos succeeded in explaining and communicating their complex technical setup and innovative method, which was developed together with Adidas, in an understandable way that you could also implement at home.
+After all of this valuable input, I had the chance to try it out and experiment with it myself.
+For the first time in ages, I had a pair of scissors and tape in my hands to create something artistic. 
+
+Before the workshop, it was said: "We believe: Ideas arise in the simplest way while making (...)
```

Comment:
The quote is from the website just fyi :) https://tomorrowacademy.org/graphic-design-innovation-lab

Reply:
It is not a real quote since we translated it into EN. I don't think it is comparable to the Steve Jobs quote and should be treated differently. That's why I would like to stick to this less formal solution.